package.json

```diff
@@ -12,6 +12,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "ascii-table": "0.0.9",
```

Comment:
I was expecting to find something that will build directly as MD, but this was the closest thing I've found 
This library required the least modifications to achieve an MD table
I've tried these:
- https://www.npmjs.com/package/tty-table
- https://www.npmjs.com/package/console-table-printer
- https://www.npmjs.com/package/as-table